velox/docs/functions/spark/array.rst

```diff
@@ -134,13 +134,34 @@ Array Functions
 .. spark:function:: array_sort(array(E)) -> array(E)
 
     Returns an array which has the sorted order of the input array(E). The elements of array(E) must
-    be orderable. Null elements will be placed at the end of the returned array. ::
+    be orderable. Null/NaN elements will be placed at the end of the returned array, with NaN elements appearing before Null elements for float types. ::
 
         SELECT array_sort(array(1, 2, 3)); -- [1, 2, 3]
         SELECT array_sort(array(3, 2, 1)); -- [1, 2, 3]
         SELECT array_sort(array(2, 1, NULL); -- [1, 2, NULL]
```

Comment:
One `)` is missing in this example. Not your change, but could you help fix it? Thanks.